packages/runtime/test-runtime-utils/api-report/test-runtime-utils.api.md

```diff
@@ -64,6 +64,14 @@ export interface IInsecureUser extends IUser {
     name: string;
 }
 
+// @alpha (undocumented)
+export interface IInternalMockRuntimeMessage {
```

Comment:
It appears like I have to export the type of outbox message if I convert it from private to protected, not sure if we any better idea to avoid this  @Abe27342

Reply:
meh, not ideal but I'm also not super worried about it. Only concern with exposing the concept is the compat maintenance burden if we need to change it. But we overexpose in this package anyway and would like to convert it to \@fluid-private one day.